src/whylogs/app/config.py

```diff
@@ -199,6 +199,9 @@ def make_session(self, data, **kwargs):
         return SessionConfig(**data)
 
 
+logger = getLogger(__name__)
```

Comment:
Since whylogs is a library and we don't have control over when modules get imported/used, all `getLogger()` calls should be within functions or classes, not at the module level.   This makes dynamic configuration possible.   Importantly, this allows users to configure whylogs regardless of import order. This is standard with python logging for library code.

For application code which we can control the workflow of, this is not necessary, but it's probably for the best to just stick to this paradigm whylogs.